examples/entity-aggregation/README.md

```diff
@@ -115,3 +115,82 @@ Note as well that this example is not intended to be authoritative. Please see
 on writing Spark programs. You may also find
 [this reference on Spark aggregate functions](https://sparkbyexamples.com/spark/spark-sql-aggregate-functions/) helpful.
 
+## Importing customers, rentals and payments from Postgres to MarkLogic
+
+This project also contains an example Java program that uses Spark to perform a nested aggregation to accomplish the following:
+
+1. Retrieve customers from Postgres, joined with rentals which is itself joined with payments.
+2. Aggregate the payments for each rental into a "payments" column containing each payment for that rental.
+2. Aggregate the rentals (including the aggregated payments column) for each customer into a "rentals" column containing structs.
+3. Write each customer row as a new document to the "Documents" database in MarkLogic. 
+
+Each customer document in MarkLogic captures the desired data structure with rentals and related payments aggregated in
+their associated customer, as opposed to having separate customer and rental documents.
+
+Run the Spark program via the following command (the use of Gradle is solely for demonstration purposes as it
+simplifies constructing the necessary classpath for the program; in production, a user would use
+[spark-submit](https://spark.apache.org/docs/latest/submitting-applications.html) or some similar mechanism provided by their Spark environment):
+
+    ./gradlew importCustomersWithRentalsAndPayments
+
+You can then use [MarkLogic's qconsole application](https://docs.marklogic.com/guide/qconsole/intro) to view the
+customer documents written to the Documents database. In this example, 10 customer rows are queried from the Postgres
+"customers" table, so you will see 10 customer documents in the Documents database.
+
+As an example, one of these customers (#182), has a single rental (#4591) that has multiple payments. If you examine the
+document for that customer, (/customerWithDoubleNesting/182.json) you will be able to verify this nested aggregation:
+
+```
+{
+  "customer_id": 182,
+  "last_name": "Lane",
+  "Rentals w/ Payments": [
```

Comment:
Just call this "rentals", that's more realistic as I don't think it's common to ever have a "/" in a JSON field name.